export_snowflake/__init__.py

```diff
@@ -41,6 +45,15 @@
 ERROR_START_MARKER = '[target_error_start]'
 ERROR_END_MARKER = '[target_error_end]'
 
+# single s3_client and db_sync object per process
+s3_client = None
+db_sync = None
+def initialize_s3_client_db_sync(config, o, file_format_type):
+    global s3_client
+    global db_sync
```

Comment:
the s3_client and db_sync instances are shared per parallel process, but not shared across.

Reply:
Could you clarify more? Could we refactor such that there's no global vars?

Reply:
Update:
Boto3 clients are threads safe, we could instance one s3_client and share it between threads. This will make it faster compared to instance a new client in each thread.
No global objects anymore.

https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html#multithreading-or-multiprocessing-with-clients

Reply:
still seeing global here

Reply:
updated, removed the unused global params and methods.